aws/resource_aws_quicksight_group.go

```diff
@@ -0,0 +1,181 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/quicksight"
+)
+
+func resourceAwsQuickSightGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsQuickSightGroupCreate,
+		Read:   resourceAwsQuickSightGroupRead,
+		Update: resourceAwsQuickSightGroupUpdate,
+		Delete: resourceAwsQuickSightGroupDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"aws_account_id": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
This argument can be made optional by reading the AWS Account ID information from the provider itself, e.g.

```go
	awsAccountID := meta.(*AWSClient).accountid
	namespace := d.Get("namespace").(string)

	if v, ok := d.GetOk("aws_account_id"); ok {
		awsAccountID = v.(string)
	}

	createOpts := &quicksight.CreateGroupInput{
		AwsAccountId: aws.String(awsAccountID),
		Namespace:    aws.String(namespace),
		GroupName:    aws.String(d.Get("group_name").(string)),
	}
```

Then the testing can/should omit the `aws_caller_identity` data source. 👍 

```suggestion
				Optional: true,
```

Reply:
fixed.